src/Components/design/Sidebar/Sidebar.js

```diff
@@ -6,96 +6,43 @@ import {
   MdHome,
   MdLocalAtm,
   MdExtension,
-  MdBarChart,
   MdAccountBalance,
   MdFormatListBulleted,
   MdPerson,
 } from "react-icons/md";
 import "./Sidebar.css";
 import CurrentOrganizationLink from "../../Helpers/CurrentOrganizationLink";
 
+const LINKS = [
+  { icon: <MdHome />, title: "Dashboard", path: "/dashboard" },
+  { icon: <MdLocalAtm />, title: "Grants", path: "/grants" },
+  { icon: <MdExtension />, title: "Boilerplates", path: "/boilerplates" },
+  {
+    icon: <MdAccountBalance />,
+    title: "Funding Organizations",
+    path: "/funding_orgs",
+  },
+  { icon: <MdFormatListBulleted />, title: "Categories", path: "/categories" },
+  { icon: <MdPerson />, title: "Users", path: "/users" },
+];
+
```

Comment:
Did a slight refactor here that should make it quicker to add/remove links in the future. If we end up having more complex variants of links in the future we may want to go back to manually rendering each item, or providing an escape hatch.

Reply:
I love this!